app/helpers/service_provider_helper.rb

```diff
@@ -128,4 +131,56 @@ def formatted_config_hash(sp_json)
     end.to_h
   end
   # rubocop:enable Layout/LineLength
+
+  def map_oidc_display_configs(sp_hash, sp_id, agency = {name => '', id => nil})
+    mapped_hash = { 
+      'agency_id' => agency['id'],
+      'friendly_name' => sp_hash['friendly_name'],
+      'agency' => agency['name'],
+      'logo' => '<REPLACE_ME.png>',
+      'certs' => '<REPLACE_ME>',
+      'return_to_sp_url' => sp_hash['return_to_sp_url'],
+      'redirect_uris' => sp_hash['redirect_uris'],
+      'return_to_sp_url' => sp_hash['return_to_sp_url'],
+      'failure_to_proof_url' => sp_hash['failure_to_proof_url'],
+      'ial' => sp_hash['ial'],
+      'attribute_bundle' => sp_hash['attribute_bundle'],
+      'restrict_to_deploy_env' => 'prod',
+      'protocol' => sp_hash['protocol'],
```

Comment:
**issue:** follow up - the raw `identity_protocol` attribute won't be quite right since there are three options permitted (`saml`, `openid_connect_private_key_jwt`, and `openid_connect_pkce`), but we want it to either be `oidc` or `saml` exactly. What do you think about setting this attribute explicitly in each of the protocol-specific methods (e.g. overriding what comes in as part of the `config_hash`)?

Reply:
I definitely didn't read that first comment correctly.